modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,68 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Identify the labels that you want to use for selecting your nodes. For this example, select nodes with labels to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
+.. To apply a label to a node, run the following command:
++
+[source,terminal]
+----
+$ oc label node <worker_node_name> <label>=<value>
+----
++
+This labels your worker node with the `<label>` label that has a value of `<value>`.
+
+. To add a label selector, edit the `KataConfig` custom resource (CR):
++
+[source,terminal]
+----
+$ oc edit kataconfig
+----
+.Example
++
+[source,yaml]
+----
+  apiVersion: kataconfiguration.openshift.io/v1
+  kind: KataConfig
+  metadata:
+    name: cluster-kataconfig
+  spec:
+    kataConfigPoolSelector:
+      matchLabels:
+         custom-kata-machine-pool: true
+----
+. Use `oc apply -f <kata-config-with-label-selector.yaml>` to apply the label to a node.
+
+.Verification
+
+* You can check to see if the nodes in the `machine-config-pool` object are going through a config update.
+** If you are using the default nodes, you can monitor the `machine-config-pool` resource by running:
++
+[source,terminal]
+----
+$ watch oc get mcp worker
+----
+** If you are using selected nodes, you can monitor the `machine-config-pool` resource by running:
++
+[source,terminal]
+----
+$ watch oc get mcp kata-oc
+----
+
+* Running `watch oc describe kataconfig cluster-kataconfig` displays information about `sandboxed-containers` extension failure on a node. The information is gathered from the status of the `machine-config-pool` object. You can view the information by running:
```

Comment:
s/Running `watch oc describe kataconfig cluster-kataconfig` displays/You can run `watch oc describe kataconfig cluster-kataconfig` to display` to make the two bullets consistent. ?